packer/http/opensuse-leap-42.2/autoinst.xml

```diff
@@ -0,0 +1,208 @@
+
+  <networking>
+    <ipv6 config:type="boolean">false</ipv6>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+    <dns>
+      <dhcp_hostname config:type="boolean">true</dhcp_hostname>
+      <dhcp_resolv config:type="boolean">true</dhcp_resolv>
+      <domain>osuosl.org</domain>
+      <hostname>opensuse-leap-42.2-x64</hostname>
+    </dns>
+    <interfaces config:type="list">
+      <interface>
+        <bootproto>dhcp</bootproto>
+        <device>eth0</device>
+        <startmode>onboot</startmode>
+      </interface>
+    </interfaces>
+  </networking>
+
+  <firewall>
+    <enable_firewall config:type="boolean">true</enable_firewall>
+    <start_firewall config:type="boolean">true</start_firewall>
+  </firewall>
+
+  <software>
+    <image/>
+    <instsource/>
+    <packages config:type="list">
+      <package>autoyast2</package>
+      <package>curl</package>
+      <package>e2fsprogs</package>
+      <package>glibc-locale</package>
+      <package>glibc</package>
+      <package>grub2</package>
+      <package>iputils</package>
+      <package>kernel-default</package>
+      <package>kexec-tools</package>
+      <package>ntp</package>
+      <package>openssh</package>
+      <package>openSUSE-release</package>
+      <package>sudo</package>
+      <package>wget</package>
+      <package>yast2-firstboot</package>
+      <package>yast2</package>
+      <package>yast2-trans-en_US</package>
+      <package>zypper</package>
+    </packages>
+    <patterns config:type="list">
+      <pattern>sw_management</pattern>
+      <pattern>yast2_install_wf</pattern>
+      <pattern>minimal_base</pattern>
+    </patterns>
+    <remove-packages config:type="list">
+      <package>telnet</package>
+    </remove-packages>
+  </software>
+  <services-manager>
+    <default_target>graphical</default_target>
+    <services>
+      <disable config:type="list"/>
+      <enable config:type="list">
+        <service>sshd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <groups config:type="list">
+    <group>
+      <gid>100</gid>
+      <groupname>users</groupname>
+      <userlist/>
+    </group>
+  </groups>
+  <user_defaults>
+    <expire/>
+    <group>100</group>
+    <groups/>
+    <home>/home</home>
+    <inactive>-1</inactive>
+    <no_groups config:type="boolean">true</no_groups>
+    <shell>/bin/bash</shell>
+    <skel>/etc/skel</skel>
+    <umask>022</umask>
+  </user_defaults>
+
+  <users config:type="list">
+    <user>
+      <user_password>opensuse</user_password>
+      <username>root</username>
+    </user>
+    <user>
+      <fullname>opensuse</fullname>
+      <gid>100</gid>
+      <home>/home/opensuse</home>
+      <password_settings>
+        <expire/>
+        <flag/>
+        <inact>-1</inact>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>opensuse</user_password>
```

Comment:
Yes, but cloud-init will lock the user on boot. This follows the same standard we do on all of the other vms.